app/Http/Controllers/CampaignRoleController.php

```diff
@@ -306,7 +304,7 @@ public function bulk()
             }
 
             if ($action === 'delete' && !$plugin->isAdmin() && !$plugin->isPublic()) {
-                $this->destroy($plugin, true);
+                $plugin->delete();
```

Comment:
Just noticed that the variable is still named `$plugin` instead of what it is :)